test/powershell/Language/Parser/Parser.Tests.ps1

```diff
@@ -71,10 +71,10 @@ Describe "ParserTests (admin\monad\tests\monad\src\engine\core\ParserTests.cs)"
 		$functionDefinition>$functionDefinitionFile
 
         $PowerShell = [powershell]::Create()
+        $PowerShell.AddScript(". $functionDefinitionFile").Invoke()

```

Comment:
ah, I see you were talking about the diff.
It's useful to also see the relative numbers as well as absolute.